subsys/bluetooth/controller/ll_sw/ull_df.c

```diff
@@ -808,4 +790,137 @@ static uint8_t set_cte_info_in_per_adv_chain(struct ll_adv_set *adv, struct lll_
 
 	return 0;
 }
+
+#if defined(CONFIG_BT_CTLR_DF_PER_ADV_CTE_NUM_MAX) && (CONFIG_BT_CTLR_DF_PER_ADV_CTE_NUM_MAX > 1)
+static bool pdu_ext_adv_is_empty_without_cte(const struct pdu_adv *pdu)
+{
+	if (pdu->len != PDU_AC_PAYLOAD_SIZE_MIN) {
+		if ((pdu->adv_ext_ind.ext_hdr_len + PDU_AC_EXT_HEADER_SIZE_MIN) != pdu->len) {
+			/* There are adv. data in PDU */
+			return false;
+		}
+
+		const struct pdu_adv_ext_hdr *ext_hdr;
+		uint8_t size_rem = 0;
```

Comment:
Not sure if this is an unwritten rule in the controller as well, but typically in the host (at least) we declare variables at top of scope.

Reply:
Moved to the top of the scope